pkg/pillar/devicenetwork/dnc.go

```diff
@@ -205,19 +205,19 @@ func VerifyPending(pending *DPCPending,
 	// Check if all the ports in the config are out of pciBack.
 	// If yes, apply config.
 	// If not, wait for all the ports to come out of PCIBack.
-	portInPciBack, portName, usedByUUID := pending.PendDPC.IsAnyPortInPciBack(aa)
+	portInPciBack, ifName, usedByUUID := pending.PendDPC.IsAnyPortInPciBack(aa)
 	if portInPciBack {
 		if usedByUUID != nilUUID {
-			errStr := fmt.Sprintf("port %s in PCIBack "+
-				"used by %s", portName, usedByUUID.String())
-			log.Errorf("VerifyPending: %s\n", errStr)
-			pending.PendDPC.LastError = errStr
-			pending.PendDPC.LastFailed = time.Now()
+			err := fmt.Errorf("port %s in PCIBack "+
+				"used by %s", ifName, usedByUUID.String())
+			log.Errorf("VerifyPending: %s\n", err)
+			pending.PendDPC.SetPortErrorByIfname(ifName, err)
```

Comment:
This semantic change was hidden in all the syntactic reformatting of errors, and I think it is incorrect to set an errors in PendDPC since 1) nothing clears them and 2) nothing reports them because the errors are reported from the Verify of the current DPC; not for the pending.

Reply:
This gets cleared when the PendingDpc actually gets to VerifyDeviceNetworkStatus - isn't it?

	pending.PendDPC.SetPortErrorsFromIntfErrMap(intfErrMap)

This is where it resets the error?

Reply:
My Understanding is - 

1) Pending DPC is part of DPCL and report back the entire DPCL
2) If Current index != 0 --- Cloud looks to see Entry 0 to see which ports has errors
     -- These become the errors to be displayed to the user.

Is this not the place where the error would get set?

Reply:
As we discussed, the issue is elsewhere in assuming that all interfaces are tested each time around, when both the verification of a new DPC as well as the periodic verify only runs until they've found one working interface.